api/v1alpha1/odfinfoconfig.go

```diff
@@ -18,6 +18,8 @@ package v1alpha1
 
 import "k8s.io/apimachinery/pkg/types"
 
+const OcsApiEndpointExportedAddress = "ocs.openshift.io/ocs-api-endpoint-exported-address"
```

Comment:
How about this?
```suggestion
const OcsApiEndpointExportedAddress = "ocs.openshift.io/api-public-address"
```

Reply:
I think we should keep ocs in the key, maybe `ocs.openshift.io/ocs-api-public-address`?
I will wait for suggestions from @nb-ohad and then push the final annotation key.

Reply:
You already have `ocs.openshift.io/` in the name. What good does the additional `ocs-` do? It's redundant.

Reply:
I agree we should drop the ocs part but then we get api-endpoint which is kind of confusing. 
Can we change this to `ocs.openshift.io/api-server-exported-address` instead?